test/ansible-inventory/inventory/hosts

```diff
@@ -1,2 +1,5 @@
 [test]
-127.0.0.1 ansible_connection=local
+ansible_host=127.0.0.1 ansible_connection=local
```

Comment:
If you do this I think you need to put a hostname at the beginning of the line (ie `test_host ansible_host=...`), I'm not exactly sure how Ansible will process a line that only declares variables without declaring a host, as far as I can tell from the docs the behavior isn't well-defined.

Reply:
Is not the host declared on top [test] ?

Reply:
Hi @fabianvf it shows not required. 

> You can also select the connection type and user on a per host basis:
> 
> [targets]
> 
> localhost              ansible_connection=local
> other1.example.com     ansible_connection=ssh        ansible_user=mpdehaan
> other2.example.com     ansible_connection=ssh        ansible_user=mdehaan

See : https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html

Reply:
The declaration at the top is for the group, which can include one or more hosts. Each line under the group declares a host, and you can provide some variables to be applied per-host. Even if it works now I'm not sure the behavior is guaranteed to be consistent, and especially if you added additional hosts to the group I think it could cause issues.

In the example posted, the pattern is `host var1=val1 var2=val2`, not the variables by themselves.

Reply:
I am not sure if I got. However, I think it no matter so has no reason for not do the change that you are requesting here :-)

Reply:
Done